src/main/java/com/snack/news/service/SlackBotService.java

```diff
@@ -0,0 +1,59 @@
+package com.snack.news.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.snack.news.domain.slack.SlackChannel;
+import com.snack.news.repository.SlackBotRepository;
+import com.snack.news.util.SlackAuthHttpClient;
+
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@ConfigurationProperties("slackbot")
+@Setter
+public class SlackBotService {
+	private final SlackBotRepository slackBotRepository;
+	private final SlackAuthHttpClient slackAuthHttpClient;
+
+	public SlackBotService(SlackBotRepository slackBotRepository, SlackAuthHttpClient slackAuthHttpClient) {
+		this.slackBotRepository = slackBotRepository;
+		this.slackAuthHttpClient = slackAuthHttpClient;
+	}
+
+	private String slackAuthUrl;
+	private String clientId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)